Makefile

```
@@ -156,3 +156,6 @@ lint-umbrella-charts:
 	ct lint --target-branch release-2.3 --chart-dirs .
```

Can you change this to `main`

```suggestion
	ct lint --target-branch main --chart-dirs .
```